virtinst/install/unattended.py

```diff
@@ -353,3 +354,17 @@ def _get_installation_source(os_media):
         script.set_config(config)
         scripts.append(script)
     return scripts
+
+
+def download_drivers(locations, tempdir):
```

Comment:
Using urlfetcher here will give us meter support, and http mocking. grep for DirectFetcher init and usage in installtreemedia.py, it can be used for downloading full URL paths by passing fullurl to acquireFile callback. You can use that to implement this function instead